week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -7,8 +7,15 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
 
-function eligibleStudents() {
-
+function eligibleStudents(studentsname) {
+  let eligibleOnes = [];
+  for(let i = 0 ;i < studentsname.length; i++){
```

Comment:
It'd be better if you used a `filter` here so that you use what you learnt in the lesson :)